seed/challenges/01-front-end-development-certification/basic-bonfires.json

```diff
@@ -374,7 +374,9 @@
         "assert(confirmEnding(\"He has to give me a new name\", \"name\") === true, 'message: <code>confirmEnding(\"He has to give me a new name\", \"name\")</code> should return true.');",
         "assert(confirmEnding(\"Open sesame\", \"same\") === true, 'message: <code>confirmEnding(\"Open sesame\", \"same\")</code> should return true.');",
         "assert(confirmEnding(\"Open sesame\", \"pen\") === false, 'message: <code>confirmEnding(\"Open sesame\", \"pen\")</code> should return false.');",
+        "assert(confirmEnding(\"Open sesame\", \"game\") === false, 'message: <code>confirmEnding(\"Open sesame\", \"same\")</code> should return true.');",
```

Comment:
I think you just forgot to update the message 😄 When you change it to return false, can you also change the `same` to `game` in the message like it is in the test? Thank you for your contribution!